examples/lammps/h2o-cavmd/README.md

```diff
@@ -0,0 +1,39 @@
+Cavity molecular dynamics of liquid water under vibrational strong coupling
+=======================
+
+This runs a cavity molecular dynamics (CavMD) simulation of liquid water under vibrational strong coupling. 
+
+In this simulation, an infrared cavity mode at 3550 cm<sup>-1</sup> is strongly coupled to the O-H stretch mode of liquid water along both the x and y directions. Due to this strong coupling, the O-H stretch band of liquid water is splitted to two eigenstates, known as polaritons.
+
+See the paper [Proc. Natl. Acad. Sci., 2020, 117 (31), 18324–18331](https://doi.org/10.1073/pnas.2009272117) or [https://github.com/TaoELi/cavity-md-ipi](https://github.com/TaoELi/cavity-md-ipi) for more details of CavMD.
+
+This example is the same as the [Rabi splitting tutorial](https://github.com/TaoELi/cavity-md-ipi/tree/master/tutorials/Rabi_splitting) in [https://github.com/TaoELi/cavity-md-ipi](https://github.com/TaoELi/cavity-md-ipi). The only difference is that in the original [Rabi splitting tutorial](https://github.com/TaoELi/cavity-md-ipi/tree/master/tutorials/Rabi_splitting), the parameters for the photons are controlled by a separate input file **photon_params.json**. Here, instead, the photon parameters are controlled in the **ffcavphsocket** section of the i-pi input file **input.xml**. 
+
```

Comment:
I would suggest adding something along the lines:

"Important: At the moment this implementation assumes that the total dipole moment is a sum of atomic charges which are fixed during the simulation while an extension to use on-the-fly dipole moments obtained from the drive is possible, it has not been implemented at the moment. If you are interested in this feature please contact us.